src/CalcViewModel/CalcViewModel.vcxproj.filters

```diff
@@ -1,28 +1,25 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
+    <Filter Include="Resources">
```

Comment:
nit: the "Resources" filter can be removed

Reply:
done - removed from CalcViewModel and CalcViewModelCopyForUT